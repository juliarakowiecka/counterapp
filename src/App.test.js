import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import {render} from '@testing-library/react'
import App from './App';


it('renders without crashing', () => {
  shallow(<App />);
});

it('renders without crashing', () => {
  const div = document.createElement("div");
  ReactDOM.render(<App />, div);
  })
 
 it('should take a snapshot', () => {
    const { asFragment } = render(<App />)
    expect(asFragment(<App />)).toMatchSnapshot()
});
