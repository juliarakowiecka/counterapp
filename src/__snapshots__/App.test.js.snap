// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <header
      class="App-header"
    />
  </div>
</DocumentFragment>
`;
